servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/ClientSecurityConfig.java

```diff
@@ -256,7 +256,7 @@ public ClientSslConfig asSslConfig() {
         }
 
         if (hostnameVerificationAlgorithm == null) {
-            builder.disableHostnameVerification();
+            builder.hostnameVerificationAlgorithm("");
```

Comment:
Can you please add at the end of `@deprecated` description of `ClientSslConfigBuilder#disableHostnameVerification()` that if users are sure they need to disable it, they can use `hostnameVerificationAlgorithm("")`.

Reply:
Done.